modules/datahandling/projectrequests.go

```diff
@@ -417,6 +414,24 @@ func (p projectLookupRequest) process(db dbfs.DBFS) ([]dhClosure, error) {
 	return []dhClosure{toSenderClosure{msg: res}}, nil
 }
 
+func projectLookup(senderID string, projectID int64, db dbfs.DBFS) (projectLookupResult, error) {
+	var resultData projectLookupResult
```

Comment:
what should I return for the projectLookupResult on the first error check then? just nil?